spec/controllers/offers_controller_spec.rb

```diff
@@ -60,7 +60,7 @@
       login(another_member.user)
 
       get "index", q: offer.title.split(/\s/).first
-      expect(assigns(:offers)).to eq([offer])
+      expect(assigns(:offers).to_a).to eq([offer])
```

Comment:
It returns a proxy from ElasticSearch to a database recordset. It delegates almost everything to the underlying recordset, like `:[]`, `:to_a`, `:each` etc. But it is **not** an array itself.